test/AWS.Deploy.CLI.IntegrationTests/BeanstalkBackwardsCompatibilityTests/ExistingWindowsEnvironment/CLITests.cs

```diff
@@ -0,0 +1,108 @@
+{
+    [Collection(nameof(WindowsTestContextFixture))]
+    public class CLITests
+    {
+        private readonly WindowsTestContextFixture _fixture;
+        private readonly AWSElasticBeanstalkHandler _awsElasticBeanstalkHandler;
+        private readonly Mock<IOptionSettingHandler> _optionSettingHandler;
+        private readonly ProjectDefinitionParser _projectDefinitionParser;
+        private readonly RecipeDefinition _recipeDefinition;
+
+        public CLITests(WindowsTestContextFixture fixture)
+        {
+            _projectDefinitionParser = new ProjectDefinitionParser(new FileManager(), new DirectoryManager());
+            _fixture = fixture;
+            _optionSettingHandler = new Mock<IOptionSettingHandler>();
+            _awsElasticBeanstalkHandler = new AWSElasticBeanstalkHandler(null, null, null, _optionSettingHandler.Object);
+            _recipeDefinition = new Mock<RecipeDefinition>(
+                It.IsAny<string>(),
+                It.IsAny<string>(),
+                It.IsAny<string>(),
+                It.IsAny<DeploymentTypes>(),
+                It.IsAny<DeploymentBundleTypes>(),
+                It.IsAny<string>(),
+                It.IsAny<string>(),
+                It.IsAny<string>(),
+                It.IsAny<string>(),
+                It.IsAny<string>()).Object;
+        }
+
+        [Fact]
+        public async Task DeployToExistingBeanstalkEnvironment()
+        {
+            var projectPath = _fixture.TestAppManager.GetProjectPath(Path.Combine("testapps", "WebAppNoDockerFile", "WebAppNoDockerFile.csproj"));
+            var deployArgs = new[] { "deploy", "--project-path", projectPath, "--application-name", _fixture.EnvironmentName, "--diagnostics", "--silent", "--region", "us-west-2" };
+            Assert.Equal(CommandReturnCodes.SUCCESS, await _fixture.App.Run(deployArgs));
+
+            var environmentDescription = await _fixture.AWSResourceQueryer.DescribeElasticBeanstalkEnvironment(_fixture.EnvironmentName);
+
+            // URL could take few more minutes to come live, therefore, we want to wait and keep trying for a specified timeout
+            await _fixture.HttpHelper.WaitUntilSuccessStatusCode(environmentDescription.CNAME, TimeSpan.FromSeconds(5), TimeSpan.FromMinutes(5));
+
+            var successMessagePrefix = $"The Elastic Beanstalk Environment {_fixture.EnvironmentName} has been successfully updated";
+            var deployStdOutput = _fixture.InteractiveService.StdOutReader.ReadAllLines();
+            var successMessage = deployStdOutput.First(line => line.Trim().StartsWith(successMessagePrefix));
+            Assert.False(string.IsNullOrEmpty(successMessage));
+
+            var expectedVersionLabel = successMessage.Split(" ").Last();
+            Assert.True(await _fixture.EBHelper.VerifyEnvironmentVersionLabel(_fixture.EnvironmentName, expectedVersionLabel));
+        }
+
+        [Fact]
+        public async Task SetupWindowsDeploymentManifestTest()
```

Comment:
This test should be place here - https://github.com/aws/aws-dotnet-deploy/blob/main/test/AWS.Deploy.Orchestration.UnitTests/ElasticBeanstalkHandlerTests.cs

Reply:
done